services/galley/src/Galley/API/Error.hs

```diff
@@ -119,6 +119,7 @@ legalHoldServiceUnavailable = Error status400 "legalhold-unavailable" "legal hol
 legalHoldServiceBadResponse :: Error
 legalHoldServiceBadResponse = Error status400 "legalhold-status-bad" "legal hold service: invalid status response"
 
+-- TODO rename to 'legalHoldNotConfigured'
```

Comment:
Now might be a good time to do this TODO :P 

If not that's fine too.

Reply:
tuns out we're not throwing this any more...  :-)